tests/policies/test_replay_buffer_wrapper.py

```diff
@@ -32,21 +32,20 @@ def make_algo_with_wrapped_buffer(
     buffer_size: int = 100,
 ) -> off_policy_algorithm.OffPolicyAlgorithm:
     venv = util.make_vec_env("Pendulum-v1", n_envs=1)
-    rl_kwargs = dict(
-        replay_buffer_class=ReplayBufferRewardWrapper,
-        replay_buffer_kwargs=dict(
-            replay_buffer_class=replay_buffer_class,
-            reward_fn=zero_reward_fn,
-        ),
-        buffer_size=buffer_size,
-    )
     rl_algo = rl_cls(
         policy=policy_cls,
         policy_kwargs=dict(),
         env=venv,
         seed=42,
-        **rl_kwargs,
-    )
+        # we ignore the type below because sb3 has a bug (forgot to put Type[...])
```

Comment:
Make this a TODO?

Good to have some easy way to find all these things we need to strip out once the upstream bugs get fixed, and grepping for TODO(juan) seems like a decent way to do it.